addons/mail/static/src/new/composer/emoji_picker.js

```diff
@@ -109,6 +108,13 @@ export class EmojiPicker extends Component {
             this.props.close();
         }
     }
+
+    refreshCurrentHighlightedCategory() {
+        const coords = this.gridRef.el.getBoundingClientRect(); //Get coords of the emoji-picker-content element
+        const res = document.elementFromPoint(coords.x, coords.y); //Gets the top-most element, emojis are always the top-most elements
+        //The 20px of buffer are used to make the result more acccurate to the currently viewed category
+        this.state.categoryId = parseInt(res.dataset.category);
```

Comment:
Remove all comments, they are either useless or wrong (20px buffer doesn't exist, and we don't want that)